language/move-lang/tests/move_check/expansion/named_address_assigned_multiple_times_invalid.move

```diff
@@ -1,3 +0,0 @@
-// Named addresses can only have one unique value
-address A = 0x1;
```

Comment:
Maybe I'll see it later, but is there some form of test documenting this syntax is now invalid?

Reply:
Ahh good call -- forgot to add in tests making sure this is no longer accepted. Will add now.